tool/tctl/common/tctl.go

```diff
@@ -112,6 +116,9 @@ func Run(commands []CLICommand, loadConfigExt LoadConfigFn) {
 		StringVar(&ccf.IdentityFilePath)
 	app.Flag("insecure", "When specifying a proxy address in --auth-server, do not verify its TLS certificate. Danger: any data you send can be intercepted or modified by an attacker.").
 		BoolVar(&ccf.Insecure)
+	app.Flag("ignore-config", fmt.Sprintf("Ignore Teleport config [%v] if present on disk.", defaults.ConfigFilePath)).
```

Comment:
Tbh, I think we should settle on the way to handle configuration. This rushes to add yet another option with a small immediate effect.
It is more in the direction of making tctl not assume it is running on an auth server node. As such, there might be other different changes that go along with that.

1. For example, tctl can read configuration from the working directory like git. This is not a super helpful option and will probably not be implemented but it also gets rid of the central configuration location.
1. Another idea is opt-out from defaults and make configuration mandatory - i.e. provide path to the configuration file on command line. Also with drawbacks but it might be combined with Option 1 resulting in a more favorable variation
1. Another idea is using the environment for defaults - as with the recent suggestions on `tsh env`, this can set up environment to use per shell instance. Also comes with drawbacks.